pandas/tools/tests/test_tile.py

```diff
@@ -283,6 +284,16 @@ def test_single_bin(self):
         result = cut(s, 1, labels=False)
         tm.assert_series_equal(result, expected)
 
+    def test_datetime_cut(self):
+        data = to_datetime(Series(['2013-01-01', '2013-01-02', '2013-01-03']))
+        result = cut(data, 3)
+        self.assertEqual(result[0],
+                         '(2012-12-31 23:57:07.200000, 2013-01-01 16:00:00]')
+        self.assertEqual(result[1],
+                         '(2013-01-01 16:00:00, 2013-01-02 08:00:00]')
+        self.assertEqual(result[2],
+                         '(2013-01-02 08:00:00, 2013-01-03 00:00:00]')
```

Comment:
Instead of checking each element, it is better to construct the series, and then use `assert_series_equal`:

```
exp = Series(['(2012-12-31 23:57:07.200000, 2013-01-01 16:00:00]', '(2013-01-01 16:00:00, 2013-01-02 08:00:00]', '(2013-01-02 08:00:00, 2013-01-03 00:00:00]')])
tm.assert_series_equal(result, exp)
```